app/views/components/_topic-list.html.erb

```diff
@@ -19,7 +19,7 @@
             item[:path],
             data: item[:data_attributes],
             "aria-label": item[:aria_label],
-            class: "govuk-link app-c-topic-list__link app-c-topic-list__link--no-underline #{brand_helper.color_class}"
+            class: "govuk-link govuk-link--no-underline app-c-topic-list__link #{brand_helper.color_class}"
```

Comment:
Might be out of scope but - can we remove this? `govuk-link--no-underline`
I feel like [these](https://www.gov.uk/topic/environmental-management) should have an underline as they're [links](https://design-system.service.gov.uk/styles/typography/#lists)